modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc

```diff
@@ -0,0 +1,78 @@
+= Amazon S3 Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install Amazon S3
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
```

Comment:
Please use the latest steps from the template:

. In Studio, create a Mule Project.
. Click the Exchange *(X)* icon in the Studio task bar.
. In Exchange, click *Login* and supply your Anypoint Platform username and password.
. In Exchange, select *All assets* and search for "<connector>".
. Select <connector>, click *Add to project*.
. Follow the prompts to install the connector.